sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs

```diff
@@ -24,5 +27,10 @@ public partial class LogsQueryResult
         /// Returns the query statistics if the <see cref="LogsQueryOptions.IncludeStatistics"/> is set to <c>true</c>.
         /// </summary>
         public BinaryData Statistics => _statistics.ValueKind == JsonValueKind.Undefined ? null : new BinaryData(_statistics.ToString());
+
+        /// <summary>
+        /// Returns the query visualization if the <see cref="LogsQueryOptions.IncludeVisualization"/> is set to <c>true</c>.
```

Comment:
What does it return if IncludeVisualization is false?

Reply:
Null. I think what you are trying to say is that I should add that to XML doc. 🤔

Reply:
> Null. I think what you are trying to say is that I should add that to XML doc. 🤔

![](https://media.giphy.com/media/S6HEz4XChX8boa5uuo/giphy.gif)